cfspopcon/formulas/fluxes.py

```diff
@@ -0,0 +1,111 @@
+"""Resistive flux consumption, inductive flux consumption (internally and externally on the plasma surface) during purely ohmic ramp-up."""
+import numpy as np
+from scipy import constants  # type: ignore[import]
+
+from ..unit_handling import Unitfull, ureg, wraps_ufunc
+
+
+@wraps_ufunc(
+    input_units=dict(plasma_current=ureg.A, major_radius=ureg.m, ejima_coefficient=ureg.dimensionless),
+    return_units=dict(flux_res=ureg.weber),
+)
+def calc_flux_res(plasma_current: float, major_radius: float, ejima_coefficient: float = 0.4) -> float:
+    """Calculate the resistive flux.
+
+    “PROCESS”: a systems code for fusion power plants - Part 1: Physics :cite:`process`
+    NOTE: CE, the Ejima coefficient, is chosen by default to be 0.4, See for example...
+    Chapter 8: Plasma operation and control: cite:`Gribov_2007`
+    Ohmic flux consumption during initial operation of the NSTX spherical torus :cite:`Menard_2001`
+
+    Args:
+        plasma_current: [A] :term:`glossary link<plasma_current>`
+        major_radius: [m] :term:`glossary link<major_radius>`
+        ejima_coefficient: [~] :term:`glossary link<ejima_coefficient>`
+
+
+    Returns:
+        [weber] :term:`glossary link<resistive_flux>`
+    """
+    return float(ejima_coefficient * constants.mu_0 * plasma_current * major_radius)
+
+
+### COMPONENTS OF INDUCTIVE FLUX: INTERNAL AND EXTERNAL ###
+
+
+@wraps_ufunc(input_units=dict(plasma_current=ureg.A, internal_inductance=ureg.henry), return_units=dict(internal_flux=ureg.weber))
+def calc_flux_internal(plasma_current: float, internal_inductance: float) -> Unitfull:
+    """Calculate the flux due to the plasma current and internal inductance of the plasma (assuming a circular cross-section).
+
+    A power-balance model for local helicity injection startup in a spherical tokamak :cite:`Barr_2018`
+    NOTE: This is (plasma current times) equation 25 from Barr but applied to a plasma with a
+    circular cross-section and a non-cirular cross-section.
+
+    Args:
+        plasma_current: [A] :term:`glossary link<plasma_current>`
+        internal_inductance: [henry] :term:`glossary link<internal_inductance>`
+
+    Returns:
+        [weber] :term:`glossary link<internal_flux>`
```

Comment:
Would you mind double checking that all returns are written as below: 

```suggestion
        [weber] :term:`internal_flux`
```